src/table.rs

```diff
@@ -54,6 +58,8 @@
 // TOMBSTONE - Deleted entry marker. 0xffff
 // NEXT - 64-bit index of the next deleted entry.
 
+// [Review] for big value, would it be an idea to directly store a file instead of miltiparting.
+// Like using NEXT pointing to 0 (metadata).
```

Comment:
I'm afraid this can lead to lots of files in some extreme cases.Typically we don't need that much performance when reading/writing big blobs (think. block bodies)